generate_jst_sh_connectors.py

```diff
@@ -697,7 +706,7 @@ def generate_jst(
 
 if __name__ == "__main__":
 
-    create_date = '2024-05-03T17:19:09Z'
+    create_date = '2024-05-30T18:30:11Z'
```

Comment:
This is the initial creation date, so this change should be reverted.

```suggestion
    create_date = '2024-05-03T17:19:09Z'
```